jdi-light-angular-tests/src/main/java/io/github/com/pages/sections/BottomSheetSection.java

```diff
@@ -0,0 +1,11 @@
+package io.github.com.pages.sections;
+
+import com.epam.jdi.light.angular.elements.complex.BottomSheet;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+
+public class BottomSheetSection {
```

Comment:
The BottomSheetSection is not required at all, given that there is only one Angular BottomSheet element.